Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_deleted_multiple_users_for_the_first_time.yml

```diff
@@ -0,0 +1,1113 @@
+    task:
+      id: 5a1c3674-24e0-4673-8ff5-f8cccc20b8e7
+      version: -1
+      name: Disable the user account
+      description: Disables an Active Directory user account.
+      script: '|||ad-disable-account'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      username:
+        simple: ${Core.OriginalAlert.stateful_raw_data.normalized_evtlog_subject_user}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1077,
+          "y": 1430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 9b7484ec-8561-4f5c-8a6a-e447014bffbe
+    type: condition
+    task:
+      id: 9b7484ec-8561-4f5c-8a6a-e447014bffbe
+      version: -1
+      name: Evaluate investigation findings
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      True Possitive:
+      - "21"
+    separatecontext: false
+    conditions:
+    - label: True Possitive
+      condition:
+      - - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: Evidence
+                transformers:
+                - operator: DT
+                  args:
+                    dt:
+                      value:
+                        simple: '.=Array.isArray(val) ? val : Object.values(val)'
+                - operator: count
+            iscontext: true
+          right:
+            value:
+              simple: "2"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 685,
+          "y": 1013
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: db53c176-f9d8-4677-ad2e-2107e9c24a69
+    type: regular
+    task:
+      id: db53c176-f9d8-4677-ad2e-2107e9c24a69
+      version: -1
+      name: Set risky host
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
```

Comment:
What about XSIAM?
 For Cortex XSIAM, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Create-a-script